modules/ddl-and-loading/pages/creating-a-loading-job.adoc

```diff
@@ -108,6 +108,72 @@ RUN LOADING JOB job1 USING file1="m1:/data/v1_1.csv", file2="m2:/data/e2.csv" <2
 <1> File path specified at compile time.
 <2> Run-time specification will override path specified at compile time.
 
+=== Loading NULL Values
```

Comment:
We already have sections that talk about how to load each type of data. It is called "Attributes and attribute expressions", with a subsection for each data type.
I would move this section there, and follow the format used there.